Code/GraphMol/ChemReactions/testReaction.cpp

```diff
@@ -5248,7 +5248,7 @@ void test54RedundantProductMappingNumbersAndRSChirality() {
     TEST_ASSERT(prods[0].size() == 1);
 
     std::cout << MolToSmiles(*prods[0][0], true) << std::endl;
-    smi = "F[C@@](Cl)(Br)ONO[C@@](F)(Cl)Br";
+    smi = "F[C@](Cl)(Br)ONO[C@](F)(Cl)Br";
```

Comment:
I'm just going to assume that the chirality changes are correct.

Reply:
(or canonicalization changes)

Reply:
Although I'm confused here why the atoms are reordered but the chirality flips.

Reply:
The structure has a mirror plane, so it's not actually chiral at all. Either `@` or `@@` is correct, it's just important to be consistent

Reply:
Arguably `assignStereochemistry()` should remove the chirality in situations like this, but that would definitely be a different PR.